disnake/ui/action_row.py

```diff
@@ -451,7 +464,11 @@ def with_message_components(cls) -> ActionRow[MessageUIComponent]:
         return ActionRow[MessageUIComponent]()
 
     @classmethod
-    def from_message(cls, message: Message) -> List[ActionRow[MessageUIComponent]]:
+    def rows_from_message(  # TODO: Name pending, may be moved altogether.
```

Comment:
this is a lot better imo. I would rather it isn't made a util method and this name seems to work for me.

@shiftinv, your thoughts?

Reply:
Personally agree that I'd rather see it on `ActionRow` than have it be some external util. I've removed the TODO comment as of dfae6bc01a31cda21cc352f5784a777c65148d13 for now while we await shift's judgment ^^

Reply:
Yeah lgtm, `rows_from_message` seems fine and having it as a classmethod is definitely preferable over a util method as well